service/src/main/java/bio/terra/workspace/service/workspace/flight/CreateGcpContextFlight.java

```diff
@@ -24,33 +25,34 @@ public CreateGcpContextFlight(FlightMap inputParameters, Object applicationConte
     FlightBeanBag appContext = FlightBeanBag.getFromObject(applicationContext);
     CrlService crl = appContext.getCrlService();
 
-    RetryRule bufferRetryRule = RetryRules.buffer();
-
     UUID workspaceId =
         UUID.fromString(inputParameters.get(WorkspaceFlightMapKeys.WORKSPACE_ID, String.class));
     AuthenticatedUserRequest userRequest =
         inputParameters.get(JobMapKeys.AUTH_USER_INFO.getKeyName(), AuthenticatedUserRequest.class);
 
-    RetryRule retryRule = RetryRules.shortExponential();
+    RetryRule dbRetry = RetryRules.shortDatabase();
+    RetryRule shortRetry = RetryRules.shortExponential();
+    RetryRule cloudRetry = RetryRules.cloud();
+
     addStep(
         new CheckSpendProfileStep(
             appContext.getWorkspaceDao(),
             appContext.getSpendProfileService(),
             workspaceId,
             userRequest));
-    addStep(new GenerateProjectIdStep());
+    addStep(new GenerateRbsRequestIdStep());
```

Comment:
Will changing this class name cause problems for any currently running/to-be-recovered V1 flights? I don't know whether Stairway serializes flight + step classes, or just the flight class.

Reply:
No. As long as the step order remains the same, renaming a step will not cause a problem.